EKF/common.h

```diff
@@ -43,6 +43,23 @@
 
 #include <matrix/math.hpp>
 
+namespace ecl{
```

Comment:
It seems to me that this would be better placed  inside `EKF/utils.hpp`

Reply:
I tried that previously and ran into a redefinition issue.

Reply:
Yes, it was missing a `#pragma once` or a classic header guard, to avoid including it multiple times into a compilation unit.
I will move it to the utils.hpp